### PR TITLE
cloud: add release notes; update roadmap

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)
- add release notes for release 20230228.
- update roadmap accordingly.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,41 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    - List Import Tasks
+    - Create Import Task
+    - Preview Data before Starting an Import Task
+    - Get Import Task Role Info
+    - Upload a Local File for an Import Task
+    - Get Import Task
+    - Update Import Task
+
+    For more information, refer to the [API documentation](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Import).
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:44]:
```suggestion
  For more information, refer to the [API documentation](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Import).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,41 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## February 28, 2023
+
+**General changes**
+
+- Add the [SQL Diagnosis](/tidb-cloud/tune-performance.md) feature for Serverless Tier clusters.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- Add the [SQL Diagnosis](/tidb-cloud/tune-performance.md) feature for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,41 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficiently. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficient. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.
```

Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficient. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 4]
```diff
@@ -8,6 +8,41 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- Release several TiDB Cloud API endpoints for the Import feature：
+
+    - List Import Tasks
+    - Create Import Task
+    - Preview Data before Starting an Import Task
+    - Get Import Task Role Info
+    - Upload a Local File for an Import Task
+    - Get Import Task
+    - Update Import Task
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:42]:
```suggestion
    - List all import tasks
    - Get an import task
    - Create an import task
    - Update an import task
    - Upload a local file for an import task
    - Preview data before starting an import task
    - Get the role information for import tasks
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 5]
```diff
@@ -8,6 +8,41 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+- Optimize the left navigation.
+
+    You can navigate pages more efficiently, for example:
+
+    - You can hover the mouse on the upper-left corner to quickly switch between clusters or projects.
+    - You can switch between the **Clusters** page and the **Admin** page.
+
+**API changes**
+
+- Release several TiDB Cloud API endpoints for the Import feature：
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:34]:
```suggestion
- Release several TiDB Cloud API endpoints for data import：
```

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 6]
```diff
@@ -8,6 +8,41 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- Upgrade the default TiDB version of all [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0) to [v6.6.0](https://docs.pingcap.com/tidb/v6.6/release-6.6.0).
+
+**Console changes**
+
+- Optimize the left navigation.
+
+    You can navigate pages more efficiently, for example:
+
+    - You can hover the mouse on the upper-left corner to quickly switch between clusters or projects.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-release-notes.md:29]:
```suggestion
    - You can hover the mouse in the upper-left corner to quickly switch between clusters or projects.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 7]
```diff
@@ -8,6 +8,41 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
 
+## February 28, 2023
+
+**General changes**
+
+- Add the [SQL Diagnosis](/tidb-cloud/tune-performance.md) feature for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
+
+    With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficient. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.
+
+    To use this feature, click **SQL Diagnosis** on the left navigation bar of your cluster page.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-release-notes.md:19]:
```suggestion
    To use SQL Diagnosis, click **SQL Diagnosis** on the left navigation bar of your Serverless Tier cluster page.
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 8]
```diff
@@ -8,6 +8,41 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+**General changes**
+
+- Add the [SQL Diagnosis](/tidb-cloud/tune-performance.md) feature for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
+
+    With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficient. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.
+
+    To use SQL Diagnosis, click **SQL Diagnosis** on the left navigation bar of your Serverless Tier cluster page.
+
+- Upgrade the default TiDB version of all [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0) to [v6.6.0](https://docs.pingcap.com/tidb/v6.6/release-6.6.0).
+
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-release-notes.md:22]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**